server.js

```diff
@@ -20,6 +21,10 @@ const METADATA = process.env.METADATA || "/etc/metadata.json";
 const BASE_URL = process.env.BASE_URL || "";
 const RELOAD_INTERVAL = parseInt(process.env.RELOAD_INTERVAL) || 0;
 
+program
+    .option('-r, --redis', 'Select redis to index the metadata');
+const args = program.parse(process.argv);
```

Comment:
I would actually prefer if we did all the options passing using environment variables - this makes it a bit cleaner to package it all up in a docker container later.

Reply:
let me change this.